Automattic/sensei
# Feature frontend improvements

Feature branch for frontend improvements


## Testing Instructions


1. Make sure you are on the latest pro trunk
2. Make sure you are on this feature branch of Course theme https://github.com/Automattic/themes/pull/7324
3. Should have these changes in place p6rkRX-6DA-p2#comment-6541

## Pre-Merge Checklist
